core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -197,14 +207,7 @@ class ZkUtils(val zkClient: ZkClient,
                               BrokerSequenceIdPath,
                               IsrChangeNotificationPath)
 
-  val securePersistentZkPaths = Seq(BrokerIdsPath,
```

Comment:
2. Probably just add the comment below object ZkUtils where all the path constants are defined?